README.md

```diff
@@ -162,36 +117,58 @@ The lifecycle events are:
 Calling this method will return an `unsubscribe` method. You can use it to stop listening to the event.
 
 ```js
-import wt, { SEND_COMPLETED } from '@clutter/wt'
+import wt, { SEND_COMPLETED } from "@clutter/wt";
 
-const unsub = wt('subscribe', SEND_COMPLETED, () => {
-  console.log('Analytics data sent - you\'ll only see this once');
+const unsub = wt.subscribe(SEND_COMPLETED, () => {
+  console.log("Analytics data sent - you'll only see this once");
   unsub();
 });
 ```
 
-### First Load
+# Server Implementation
 
-You may be sending a cookie back from your server. If you want to wait for an event to complete before performing actions, use the `afterFirstLoad` command.
+Events will be sent to the server by one of two methods:
 
-```js
-import wt from '@clutter/wt'
+1. A post request with URL encoded query string
+2. If the post is unsuccessful, a get request with the same query string to a pixel image
```

Comment:
out of scope for this PR, but we should probably remove the image stuff, we wanted to make some follow-ups to a past PR, see https://github.com/clutter/wt-client/pull/36#issuecomment-825701857.